test/call.c

```diff
@@ -969,3 +980,75 @@ int test_call_mediaenc(void)
 
 	return err;
 }
+
+
+int test_call_custom_headers(void) {
```

Comment:
the curly brace should be moved to the next line.

functions are written like this:

```
void function(void)
{
}
```